app/src/main/res/layout/row_workout.xml

```diff
@@ -5,9 +5,11 @@
         style="@style/Widget.MaterialComponents.CardView"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginLeft="@dimen/mtrl_card_spacing"
-        android:layout_marginTop="@dimen/mtrl_card_spacing"
-        android:layout_marginRight="@dimen/mtrl_card_spacing"
+        android:layout_marginLeft="0dp"
+        android:layout_marginTop="0dp"
+        android:layout_marginRight="0dp"
+        app:cardCornerRadius="0dp"
+        app:cardElevation="0dp"
```

Comment:
How come?

Reply:
imo, it looks better flat, but we can change that once the design is finalized for all screens. More of just implementing functionality now anyways